sagemaker-neo-notebooks/edge/windows/README.md

```diff
@@ -0,0 +1,216 @@
+
+# C++ Tutorial
+
+## Prerequisite
+
+Please review the following prerequisites tasks before proceed with tutorial
+
+### Environment Setup
+
+#### Environment Setup for Windows
+
+In order to compile DLR and tutorial dependencies, please install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) and [WinML](https://docs.microsoft.com/en-us/windows/ai/windows-ml/get-started-desktop) components. You also need to install [CppWinRT package](https://marketplace.visualstudio.com/items?itemName=CppWinRTTeam.cppwinrt101804264). 
+
+Additionally, please make sure msbuild path is setup properly for PowerShell. In order to do so, please append your msbuild path, such as C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin, to your system path.
+
+### Install CMake
+
+Please make sure CMake is installed prior start this tutorial. Both linux/windows distributions can be located here [https://cmake.org/download/]
+
+### Install AWS C++ SDK
+
+Please validate AWS C++ SDK is installed before starting this tutorial. In this sample tutorial, you'd need install AWS S3, IAM, and SageMaker service for Neo model compilation. For more information, please refer to [https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/welcome.html]. Installation guide can also be located [here](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/setup.html). CMake instruction can be found [here](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/build-cmake.html).
+
+#### Install SDK on Linux
+
+1. Checkout SDK from source `git clone https://github.com/aws/aws-sdk-cpp.git`
+2. Create build directory for aws-sdk and install through CMake
+
+``` shell
+cd aws-sdk-cpp
+mkdir build && cd build
+
+# install through cmake
+cmake .. \
+    -DBUILD_ONLY="s3;iam;sagemaker" \
+    -DCMAKE_BUILD_TYPE=Release \
+
+# install the sdk
+make
+make install
+```
+
+#### Install SDK on Windows
+
+1. Checkout SDK from source `git clone https://github.com/aws/aws-sdk-cpp.git`
+2. Create build directory for aws-sdk and install through CMake. Installed binaries can be located in C:\Program Files (x86)\aws-cpp-sdk-all
+
+``` shell
+cd aws-sdk-cpp
+mkdir build && cd build
+
+# install through cmake
+cmake ..\aws-sdk-cpp -D CMAKE_BUILD_TYPE=Release -D BUILD_ONLY="s3;iam;sagemaker" -D ENABLE_TESTING=OFF
```

Comment:
I think "..\aws-sdk-cpp" should be ".."

Reply:
This is actually mirror to AWS SDK documentation. They have their built as a separate directory rather inside aws-sdk-cpp. Hence it's `cmake ..\aws-sdk-cpp`. Nonetheless, the end goal is to install AWS SDK from the built repo.  I'd keep this as is just for consistency.